alembic/versions/cd0fbb73bf8e_capture_document_count.py

```diff
@@ -0,0 +1,25 @@
+"""Capture document count
```

Comment:
flagging this for maintainers, we currently have to make a change in the deb package repo when when we add an alembic migration, filed https://github.com/freedomofpress/securedrop-debian-packaging/issues/20 to track